### PR TITLE
ci: Validate docs urls for langchain nodes as well (no-changelog)

### PR DESCRIPTION
**Pull request by netroy**
## Summary
This PR updates the docs url validation script to now also validate urls for credentials and nodes in the new langchain package.
also, this speeds up the url validation script.

[Test Run](https://github.com/n8n-io/n8n/actions/runs/7460694505)

## Review / Merge checklist
- [x] PR title and summary are descriptive

### CODE REVIEW

File: .github/scripts/validate-docs-links.js [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 const path = require('path');
 const https = require('https');
-const glob = require('fast-glob');
+const glob = require('glob');
 const pLimit = require('p-limit');
+const Lookup = require('cacheable-lookup').default;
 
-const nodesBaseDir = path.resolve(__dirname, '../packages/nodes-base');
+const agent = new https.Agent({ keepAlive: true, keepAliveMsecs: 5000 });
+new Lookup().install(agent);
```
Comment (tomi) [inline .github/scripts/validate-docs-links.js:15]: Nice 👌 Wasn't aware of this package

File: .github/scripts/validate-docs-links.js [hunk 2]
```diff
@@ -22,21 +30,26 @@ const validateUrl = async (kind, name, documentationUrl) =>
 					port: 443,
 					path: url.pathname,
 					method: 'HEAD',
+					agent,
+				},
+				(res) => {
+					debug('\u2713', kind, name);
```
Comment (tomi) [inline .github/scripts/validate-docs-links.js:36]: What is this magic string? Could be moved to a constant with some descriptive name

**Review by tomi — DISMISSED** [review]
LGTM 💯 One small suggestion

File: .github/scripts/validate-docs-links.js (hunk 2 above)
Comment (tomi) [reply .github/scripts/validate-docs-links.js:36]: Could also use the character directly: ✓

Comment (netroy) [reply .github/scripts/validate-docs-links.js]: I still haven't gotten used to putting unicode chars or emojis directly in code. 
I guess I'm showing signs of being old.
image

**Review by tomi — APPROVED** [review]

**Pull request merged**